package.json

```diff
@@ -16,7 +16,7 @@
     "test": "make test"
   },
   "engines": {
-    "node": "8.10.0"
+    "node": "10.14.2"
```

Comment:
I'd accept ``">=10.14.2 <11"`` to get future versions of node 10, but I'm not sure how often node minor releases break things.

Reply:
Minor node releases almost never break things.

```suggestion
    "node": "^10.14.2"
```
(`^x.y.z` is&nbsp;equivalent to&nbsp;setting `>=x.y.z <(x+1)`)